### ISSUE TITLE
CQ500 External Validation Dataset | Precision, Recall, F1 Score Results

### ISSUE THREAD

**Issue by Lian-La-Fey** [charlierabea/FORTE#16]
Hello.

I generated reports using the CQ500 External Validation Dataset (from the link you shared) and used the FORTE metric to clinically measure the accuracy of these reports. Although the **`accuracy`** results were close to those in your paper, the **precision**, **recall**, and **F1 score** values I got were extremely low. I obtained values by applying **Negation Removal** are as follows. Did you measure **precision**, **recall**, and **F1 Score** metrics? If so, what were those values?

<img width="705" height="196" alt="Image" src="https://github.com/user-attachments/assets/06f47752-6d0e-44ec-a0f9-85da1c2ef267" />

**Comment by charlierabea** [comment]
Thanks for bringing this up. You’re right. We also noticed that only the overall accuracy on the CQ500 dataset looked relatively strong, while the precision, recall, and F1 scores were much less remarkable. The CQ500 cohort was used as a zero-shot external validation set for BrainGPT, and it differs quite a bit from our internal 3DBrainCT cohort in terms of case mix and report style.

In our internal review, we also found that some of the CQ500 labels weren’t fully consistent with our radiologists’ interpretations. Still, given its well-recognized publication status, we used it as an external benchmark without manual relabeling.

Our main goal was to evaluate the radiology report generation (RRG) pipeline rather than to optimize classification metrics. That said, we agree that broader external validation, ideally with additional datasets and fine-tuning on institution-specific data, would help improve performance and generalizability.

**Comment by Lzw-lukeskywalker** [comment]
Dear author, I was wondering if your training dataset is open source? Alternatively, could you recommend any relevant reference datasets?

**Comment by charlierabea** [comment]
> Dear author, I was wondering if your training dataset is open source? Alternatively, could you recommend any relevant reference datasets?

Please refer to https://github.com/charlierabea/FORTE/issues/12

There were few at the time we conducted this research. But I believe for CT, the Merlin paper (Stanford) is public now, and you can also check some RSNA challenges out.
